helper-cli/src/main/kotlin/commands/packageconfig/ExportLicenseFindingCurationsCommand.kt

```diff
@@ -0,0 +1,88 @@
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import org.ossreviewtoolkit.helper.common.RepositoryLicenseFindingCurations
+import org.ossreviewtoolkit.helper.common.findRepositories
+import org.ossreviewtoolkit.helper.common.getLicenseFindingCurationsByRepository
+import org.ossreviewtoolkit.helper.common.mergeLicenseFindingCurations
+import org.ossreviewtoolkit.helper.common.writeAsYaml
+import org.ossreviewtoolkit.model.config.PackageConfiguration
+import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.utils.expandTilde
+
+internal class ExportLicenseFindingCurationsCommand : CliktCommand(
+    help = "Export the license finding curations to a file which maps repository URLs to the license finding " +
+            "curations for the respective repository."
+) {
+    private val packageConfigurationFile by option(
+        "--package-configuration-file",
+        help = "The package configuration."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val sourceCodeDir by option(
+        "--source-code-dir",
+        help = "A directory containing the sources corresponding to the provided package configuration."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val updateOnlyExisting by option(
+        "--update-only-existing",
+        help = "If enabled, only entries are imported for which an entry already exists which differs only in terms " +
+                "of its concluded license, comment or reason."
+    ).flag()
+
+    private val licenseFindingCurationsFile by option(
+        "--license-finding-curations-file",
+        help = "The output license finding curations file."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    override fun run() {
+        val localLicenseFindingCurations = getLicenseFindingCurationsByRepository(
+            curations = packageConfigurationFile.readValue<PackageConfiguration>().licenseFindingCurations,
+            nestedRepositories = findRepositories(sourceCodeDir)
+        )
+
+        val globalLicenseFindingCurations = if (licenseFindingCurationsFile.isFile) {
+            licenseFindingCurationsFile.readValue<RepositoryLicenseFindingCurations>()
+        } else {
+            mapOf()
```

Comment:
Better: `emptyMap()`

Reply:
Or yet better (IMO), follow the approach outlined here:

https://github.com/oss-review-toolkit/ort/pull/3874/files#diff-fbfa4ac54e65cb37598fe5875b4c25eaa90f68955a06c9038f663599e983aad2R82-R83